src/main/java/schemadefiner/retrieveschema/util/StreamlinerUtil.java

```diff
@@ -0,0 +1,34 @@
+package schemadefiner.retrieveschema.util;
+
+import org.yaml.snakeyaml.Yaml;
+import org.yaml.snakeyaml.constructor.Constructor;
+import org.yaml.snakeyaml.representer.Representer;
+import schemacrawler.tools.databaseconnector.DatabaseConnectionSource;
+import schemacrawler.tools.databaseconnector.SingleUseUserCredentials;
+import schemadefiner.retrieveschema.model.Configuration;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.InputStream;
+import java.sql.Connection;
+
+public class StreamlinerUtil {
+
+    public static Configuration readConfigFromPath(String path) throws FileNotFoundException {
+        Representer representer = new Representer();
+        //this is used to neglect extra fields present in the file
+        representer.getPropertyUtils().setSkipMissingProperties(true);
```

Comment:
Since you'll be writing this file out soon, I think we should remove this skip missing and implement whatever is required to read the file without skipping anything.